_docs/samples/consul-integration.md

```diff
@@ -10,11 +10,12 @@ draft: false
 ---
 {% include home.html %}
 
-This sample deploys the Bookinfo application in a simple Docker Compose environment using Consul as the service registry, and demonstrates various features of the Istio service mesh on non-kubernetes platforms.
+This sample deploys the Bookinfo application in a simple Docker Compose environment using Consul as the service 
+registry, and demonstrates various features of the Istio service mesh on non-kubernetes platforms.
 
 ## Before you begin
 * Setup Istio by following the instructions in the [Installation guide]({{home}}/docs/setup/install-kubernetes.html).
-  You will need `kubectl` and Steps 1-4 from `Installation step` only for this Task.
+  You will need Steps 1-3 from `Installation step` only for this Task.
```

Comment:
Remove this entire thing.. Copy the three steps here. Those are generic steps and have nothing to do with k8s.